pkg/storaged/details.js

```diff
@@ -1501,8 +1467,7 @@
                 Level: level,
                 Bitmap: bitmap,
                 Degraded: degraded_message,
-                State: mdraid.Running? _("Running") : _("Not running"),
-                SyncAction: sync_action
+                State: mdraid.Running? _("Running") : _("Not running")
```

Comment:
If we add a trailing comma here like we usually do, then this could result in fewer lines changed in the future:
`State: mdraid.Running? _("Running") : _("Not running"),`